setup.py

```diff
@@ -59,6 +56,7 @@
     "WTForms<3.0.0,>=2.3.3",
     # invenio-celery conflict fix
     "celery<5.2,>=5.1.0",
+    "flask>=2.0.0"
```

Comment:
instead of this you could bump invenio-base, same as we have done in invenio-app-ils

Reply:
ah ofc, i was wondering why it was not setting flask 2 automatically :sweat_smile: